plugins/outputs/redistimeseries/redistimeseries_test.go

```diff
@@ -0,0 +1,26 @@
+package redistimeseries
```

Comment:
As there is zero testing in this PR that would get routinely run, I'd like to see an integration test.

There is an example in the [redis input](https://github.com/influxdata/telegraf/blob/master/plugins/inputs/redis/redis_test.go#L36) using testcontainers that you can copy and paste from.

Reply:
@powersj Maybe I misunderstand the testing (and likely)! But doesn't the TestConnectAndWrite in this file, with the MockMetrics cover this use case? I'd like to make sure I'm not missing something - and I'm pretty sure I am.